plugins/woocommerce/tests/php/src/Blocks/BlockTypes/BlockifiedProductDetails/test_product_details_render_with_hook_expected_result.html

```
@@ -0,0 +1,148 @@
+					<p>Reviews Body</p>
+				</div>
+			</div>
+		</div>
+		<div data-block-name="woocommerce/accordion-item" data-wc-class--is-open="state.isOpen"
+			data-wc-context="{&quot;id&quot;:&quot;woocommerce-accordion-item-4&quot;,&quot;openByDefault&quot;:false}"
+			data-wc-init="callbacks.initIsOpen"
+			class="wp-block-woocommerce-accordion-item is-layout-flow wp-block-woocommerce-accordion-item-is-layout-flow">
+			<h3
+				class="wp-block-woocommerce-accordion-header accordion-item__heading is-layout-flow wp-block-woocommerce-accordion-header-is-layout-flow">
+				<button aria-controls="woocommerce-accordion-item-4-panel" data-wc-bind--aria-expanded="state.isOpen"
+					data-wc-on--click="actions.toggle" id="woocommerce-accordion-item-4" class="accordion-item__toggle"
```

HTMLHint (1.5.0)

[error] The id value [ woocommerce-accordion-item-4 ] must be unique.

(id-unique)